server/market/orderrouter.go

```diff
@@ -593,6 +618,30 @@ func (r *OrderRouter) handleMarket(user account.AccountID, msg *msgjson.Message)
 	return nil
 }
 
+// Check the FundingCoin confirmations, and if zero, ensure the tx fee rate
+// is sufficient, > 90% of our last recorded estimate for the asset.
+func (r *OrderRouter) checkZeroConfs(dexCoin asset.FundingCoin, fundingAsset *asset.BackedAsset) *msgjson.Error {
+	// Verify that zero-conf coins are within 10% of the last known fee
+	// rate.
+	confs, err := coinConfirmations(dexCoin)
+	if err != nil {
+		log.Debugf("Confirmations error for %s coin %s: %v", fundingAsset.Symbol, dexCoin, err)
+		return msgjson.NewError(msgjson.FundingError, fmt.Sprintf("failed to verify coin %v", dexCoin))
+	}
+	if confs > 0 {
+		return nil
+	}
+	lastKnownFeeRate := r.feeSource.LastFeeRate(fundingAsset.ID)
+	feeMinimum := uint64(math.Round(float64(lastKnownFeeRate) * 0.9))
```

Comment:
0.9 seems pretty reasonable.  Let's make this a named constant and describe.

Also what do you think about a minimum difference allowed?  e.g. say the last was 15, then `feeMinimum` is 14 (round(15*0.9 = 13.5)), and it makes me wonder if that's too tight tolerance toward the smaller end.
Obviously we don't want things slowed down, but blocking incoming orders would suck?

Or if all client eventually update would there theoretically be no difference if they all use the fee suggestion from one of the various sources?